apps/native-component-list/src/screens/Image/tests/android.tsx

```diff
@@ -3,6 +3,18 @@ import { ImageTestGroup, ImageTestPropsFnInput } from '../types';
 const imageTests: ImageTestGroup = {
   name: 'Android',
   tests: [
+    {
+      name: 'accessible',
+      props: {
+        accessible: true,
+      },
+    },
+    {
+      name: 'accessibilityLabel',
+      props: {
+        accessibilityLabel: 'Test passed',
+      },
+    },
```

Comment:
I need you to add some inline description for these tests 🤔 
I'm referring to the steps you've described in the PR's body, we need something similar on the screen, because no one is going to remember how to test it 👀